src/main/java/edu/cornell/kfs/sys/batch/service/impl/CuAutoDisapproveDocumentsServiceImpl.java

```diff
@@ -40,19 +36,20 @@
 import org.kuali.kfs.krad.bo.Note;
 import org.kuali.kfs.krad.document.Document;
 import org.kuali.kfs.krad.service.KRADServiceLocator;
+import org.kuali.kfs.krad.service.NoteService;
 import org.kuali.kfs.krad.util.GlobalVariables;
 import org.kuali.kfs.krad.util.ObjectUtils;
 import org.springframework.transaction.annotation.Transactional;
 
-import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
-
 @Transactional
 public class CuAutoDisapproveDocumentsServiceImpl extends AutoDisapproveDocumentsServiceImpl {
 	private static final Logger LOG = LogManager.getLogger(CuAutoDisapproveDocumentsServiceImpl.class);
     private static final String TAB = "\t";
 
     private RouteHeaderService routeHeaderService;
     private PersonService personService;
+    private DocumentTypeService documentTypeService;
+    private NoteService noteService;

```

Comment:
Another instance where we're adding our own versions of variables that are private on the superclass. I would suggest either updating the service setters to also call the superclass setters, or temporarily overlaying the superclass to make those variables "protected" instead.

Reply:
change added